src/node/rpc/endpoint_registry.h

```diff
@@ -375,6 +354,40 @@ namespace ccf
         return set_auto_schema<typename T::In, typename T::Out>(status);
```

Comment:
Docstring is outdated a bit: "Sets the schema that the request parameters and response must comply with"